server/src/main/java/org/opensearch/client/Client.java

```diff
@@ -325,6 +333,26 @@ public interface Client extends OpenSearchClient, Releasable {
      */
     SearchScrollRequestBuilder prepareSearchScroll(String scrollId);
 
+    /**
+     * Create point in time for one or more indices
+     */
+    void createPit(CreatePitRequest createPITRequest, ActionListener<CreatePitResponse> listener);
```

Comment:
Posting @reta's comment here - https://github.com/opensearch-project/OpenSearch/pull/4406#discussion_r963846152

@bharath-techie from the API perspective, this is beaking change. Since we backport to 2.x (no breaking changes), we probably should mark this methods as default.

Reply:
@reta by marking default we can remove implementations in case of roll back ? is that the idea ?